src/config/tests.rs

```diff
@@ -1,10 +1,284 @@
 use crate::Config;
 use indoc::indoc;
-use serde_yaml::Error;
+
+extern crate serde_yaml;
+extern crate toml;
+
+#[test]
+fn test_toml_modmap_basic() {
```

Comment:
Could you put newer tests after the ones that currently exist? I want to see YAML tests first since the syntax is still primarily designed for YAML.